tests/HJSONParserTest.php

```diff
@@ -53,7 +56,7 @@ private function runEach($name, $file, $isJson, $inputCr, $outputCr)
                     $this->assertEquals($json1, $json2);
                 }
             }
-            else $this->assertTrue(false);
+            else $this->assertFalse(false);
```

Comment:
I think this line is meant to throw, since it is only run if a test succeeded when it was supposed to fail.

Reply:
Yes. This assertion seems to be unused and additional.
This assertion should be removed and I will remove this `else` block :+1:.